packages/adblocker/assets/easylist/easylist-cookie.txt

```diff
@@ -703,7 +703,6 @@
 /js/cookiePolicy?
 /js/CookiePrompter-
 /js/eprivacy.js
-/js/onetrust/*
```

Comment:
this should not be removed - it is likely filtered due to some TrackerDB integration